projects/documentation/src/components/styles.css

```
@@ -922,53 +922,40 @@ sp-link:not(:defined) {
     text-decoration: underline;
 }
 
-.demo-placement.with-demo {
+.demo-container {
     display: grid;
-    grid-template-areas:
-        'demo'
-        'links';
+    grid-template-columns: 3fr 1fr;
```

```suggestion
    grid-template-columns:  1fr auto;
```
This ensures the config column never needs to scroll horizontally.